hieradata/common.yaml

```diff
@@ -320,7 +320,7 @@ profile::buildmaster::default_tools:
     sourceURL: https://github.com/adoptium/temurin17-binaries/releases/download
   maven:
     mvn:
-      version: "3.8.2"
+      version: "3.8.3"
```

Comment:
The Apache Maven 3.8.3 tag has been placed in the GitHub repository, but I don't see Apache Maven 3.8.3 downloads from https://maven.apache.org/download.cgi .  I think this change should wait until developers can download the version and test it.